rules/aip0233/response_message_name.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0233
+
+import (
+	"fmt"
+
+	"github.com/gertd/go-pluralize"
+	"github.com/googleapis/api-linter/lint"
+	"github.com/googleapis/api-linter/rules/internal/utils"
+	"github.com/jhump/protoreflect/desc"
+)
+
+// Batch Create method should have a properly named Response message.
+var responseMessageName = &lint.MethodRule{
+	Name:   lint.NewRuleName("core", "0233", "response-message-name"),
+	OnlyIf: isBatchCreateMethod,
+	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
+		want := fmt.Sprintf("BatchCreate%sResponse", pluralize.NewClient().Plural(m.GetName()[11:]))
```

Comment:
We just need to check if the response message name is the name of the method + "Response" -- we don't need to check if it is pluralize, as it has been checked in another rule.

Reply:
+1, I already fixed that in the AIP-231 rule.

Reply:
Done.